.github/workflows/build.yml

```diff
@@ -506,6 +506,7 @@ jobs:
         - "adafruit_metro_esp32s2"
         - "artisense_rd00"
         - "atmegazero_esp32s2"
+        - "ai_thinker_esp_12k"
```

Comment:
Sorry, this is still not quite in alphabetical order. Put it before the artisense